modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccountSupplierRole.testcase

```diff
@@ -161,6 +337,1148 @@ definition {
+				orderItemQuantity = 3,
+				orderItemSku = "TESTSKU");
+		}
+
+		task ("When the supplier user creates 1 quantity of the shipment in a supplier order with 'Simple Product'") {
+			CommerceOrders.openOrdersAdmin();
+
+			var customerOrderId = selenium.getText("//div[(@class='dnd-td') and contains(string(), '$ 34.00')]//..//div[(@class='dnd-td')][1]");
+			var supplierOrderId1 = selenium.getText("//div[(@class='dnd-td') and contains(string(), '$ 3.28')]//..//div[(@class='dnd-td')][1]");
+			var supplierOrderId2 = selenium.getText("//div[(@class='dnd-td') and contains(string(), '$ 15.72')]//..//div[(@class='dnd-td')][1]");
+
+			CommerceNavigator.gotoOrderEntryViaAdmin(status = "$ 3.28");
+
+			CommerceOrders.checkoutFullWorkflow(
+				orderId = ${supplierOrderId1},
+				sku = "TESTSKU",
+				warehouse = "Italy");
+		}
+
+		task ("Then following the workflow of the shipment, the outstanding quantity of that product from the customer order is correctly decreased by 1 and equals 2") {
+			CommerceOrders.openOrdersAdmin();
+
+			CommerceNavigator.gotoOrderEntryViaAdmin(status = "$ 3.28");
+
+			CommerceOrders.checkoutFullWorkflow(
+				orderIds = ${customerOrderId},
+				outstandingQuantities = 2,
+				outstandingQuantity = "true",
+				productSkus = "TESTSKU",
+				warehouse = "Italy");
+		}
+
+		task ("When the status for the customer order is still not completed if only one supplier order is completed") {
+			CommerceOrders.openOrdersAdmin();
+
+			CommerceNavigator.gotoOrderEntryViaAdmin(status = "$ 3.28");
+
+			CommerceOrders.checkoutFullWorkflow(
+				orderId = ${supplierOrderId1},
+				orderItemQuantity = 2,
+				sku = "TESTSKU",
+				warehouse = "Italy");
+
+			CommerceOrders.openOrdersAdmin();
+
+			for (var orderStatusWithAmount : list "Processing:15.72,Partially Shipped:34.00,Completed:3.28") {
+				var orderStatus = StringUtil.extractFirst(${orderStatusWithAmount}, ":");
+				var amount = StringUtil.extractLast(${orderStatusWithAmount}, ":");
+
+				AssertElementPresent(
+					key_parameter = ${amount},
+					key_status = ${orderStatus},
+					locator1 = "CommerceOrders#ORDER_STATUS_BY_SECOND_PARAMETER");
+			}
+		}
+
+		task ("And the status for the customer order is completed if all the supplier orders are completed") {
+			CommerceOrders.openOrdersAdmin();
+
+			CommerceNavigator.gotoOrderEntryViaAdmin(status = "$ 15.72");
+
+			CommerceOrders.checkoutFullWorkflow(
+				orderId = ${supplierOrderId2},
+				sku = "MIN55861",
+				warehouse = "Italy");
+
+			CommerceOrders.openOrdersAdmin();
+
+			for (var orderStatusWithAmount : list "Completed:15.72,Completed:34.00,Completed:3.28") {
+				var orderStatus = StringUtil.extractFirst(${orderStatusWithAmount}, ":");
+				var amount = StringUtil.extractLast(${orderStatusWithAmount}, ":");
+
+				AssertElementPresent(
+					key_parameter = ${amount},
+					key_status = ${orderStatus},
+					locator1 = "CommerceOrders#ORDER_STATUS_BY_SECOND_PARAMETER");
+			}
+		}
```

Comment:
No need to assert all of this as it is pretty much covered in the order splitting tests: please keep only the parts regarding the changes made to the order by the supplier that need to be reflected to the customer order

Reply:
Why I write the test here just because the supplier user logs in and I want to assert from his side. In the order splitting tests, In order split testing, it is the administrator who makes the assertion. So I'm not sure if we need to assert again here.